pkg/network/infraconfigurators/bridge.go

```diff
@@ -116,11 +114,11 @@ func (b *BridgePodNetworkConfigurator) GenerateDHCPConfig() *cache.DHCPConfig {
 func (b *BridgePodNetworkConfigurator) getFakeBridgeIP() (string, error) {
 	ifaces := b.vmi.Spec.Domain.Devices.Interfaces
 	for i, iface := range ifaces {
-		if iface.Name == b.iface.Name {
+		if iface.Name == b.vmiSpecIface.Name {
 			return fmt.Sprintf(bridgeFakeIP, i), nil
 		}
 	}
-	return "", fmt.Errorf("ailed to generate bridge fake address for interface %s", b.iface.Name)
+	return "", fmt.Errorf("Failed to generate bridge fake address for interface %s", b.vmiSpecIface.Name)
```

Comment:
big nit: would you lowercase the "Failed" string  ?

Reply:
well whatever you decide to do, keep it consistent with line 198 `return fmt.Errorf("No gateway address..`

Reply:
done